src/remoteConnector.ts

```diff
@@ -615,7 +632,7 @@ export default class RemoteConnector extends Disposable {
 
 		const copy = 'Copy';
 		const configureSSH = 'Configure SSH';
-		const action = await vscode.window.showInformationMessage(`An SSH key is required for passwordless authentication.\nAlternatively, copy and use this password: ${maskedPassword}`, { modal: true }, copy, configureSSH);
+		const action = await vscode.window.showInformationMessage(`Couldn't find any SSH public key registered in your Gitpod account.\nAlternatively, copy and use this password: ${maskedPassword}`, { modal: true }, copy, configureSSH);
```

Comment:
`You don't have any public SSH keys in your Gitpod account. You can [add a new public key](https://gitpod.io/keys), or use a generated access token as a password.`

Reply:
We can't use markdown in modal notification

Reply:
Change it to `You don't have registered any SSH public key for this machine in your Gitpod account.\nAlternatively, copy and use this password:`